api-docs/postman-collection/event-manager.postman_collection

```diff
@@ -1,1071 +1,1041 @@
+					"value": "4",
+					"description": "",
+					"type": "text"
+				}
+			],
+			"rawModeData": "",
+			"url": "http://localhost:8000/api/events/add/1/",
+			"responses": [],
+			"pathVariableData": [],
+			"queryParams": [],
+			"headerData": [
+				{
+					"key": "X-CSRFToken",
+					"value": "VzOmeEVnwaYME3quXte5pLAgsi4wDTYIspLnIXuumgeQ2qhI1rFUoVDku99K3WfK",
+					"description": "",
+					"enabled": false
+				},
+				{
+					"key": "Content-Type",
+					"value": "application/json",
+					"description": "",
+					"enabled": false
+				},
+				{
+					"key": "Authorization",
+					"value": "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ0b2tlbl90eXBlIjoiYWNjZXNzIiwiZXhwIjoxNTMyNTEzNzk1LCJqdGkiOiI4OTQ2MmRhYTA5OTg0NTNmYmQ0ZGY0MDNjNjg1ZjBlZSIsInVzZXJfaWQiOiJhODFkYmVjNy0yNDhkLTRkZTQtOGM2NS0xOGI2ZmI0ODc1NGUifQ.Gq7cV3f_T6OG1geahBbO_-om_cahn35lH2gXqO-qWic",
+					"description": ""
+				},
+				{
+					"key": "Accept",
+					"value": "application/json",
+					"description": "",
+					"enabled": false
+				}
+			],
+			"collection_id": "d73986cb-86d7-0c45-7a74-94e20e161e76"
 		},
 		{
-			"name": "{{URI}}/api/organisations/",
-			"request": {
-				"url": "{{URI}}/api/organisations/",
-				"method": "POST",
-				"header": [
-					{
-						"key": "X-CSRFToken",
-						"value": "VzOmeEVnwaYME3quXte5pLAgsi4wDTYIspLnIXuumgeQ2qhI1rFUoVDku99K3WfK",
-						"description": "",
-						"disabled": true
-					},
-					{
-						"key": "Content-Type",
-						"value": "multipart/form-data",
-						"description": "",
-						"disabled": true
-					},
-					{
-						"key": "Authorization",
-						"value": "{{token}}",
-						"description": ""
-					},
-					{
-						"key": "application/json",
-						"value": "",
-						"description": ""
-					}
-				],
-				"body": {
-					"mode": "formdata",
-					"formdata": [
-						{
-							"key": "name",
-							"value": "Organisation 2u",
-							"description": "",
-							"type": "text"
-						},
-						{
-							"key": "owner",
-							"value": "b6e21be3-ae59-42b0-acd2-ca48690cff9b",
-							"description": "",
-							"type": "text"
-						},
-						{
-							"key": "logo",
-							"description": "",
-							"type": "file"
-						},
-						{
-							"key": "start_datetime",
-							"value": "2081-02-08T20:39:00+0000",
-							"description": "",
-							"type": "text",
-							"disabled": true
-						},
-						{
-							"key": "end_datetime",
-							"value": "3938-02-08T15:08:00+0000",
-							"description": "",
-							"type": "text",
-							"disabled": true
-						},
-						{
-							"key": "organisation",
-							"value": "1",
-							"description": "",
-							"type": "text",
-							"disabled": true
-						},
-						{
-							"key": "description",
-							"value": "test dexcriptuon",
-							"description": "",
-							"type": "text"
-						}
-					]
-				},
-				"description": ""
-			},
-			"response": []
+			"id": "2fd80832-e021-cc36-3854-8c87583f0893",
+			"name": "http://localhost:8000/api/token",
```

Comment:
@JavaEria use env configuration here .. consult @ZenabKhan for quick guide on this